Libraries/LibWeb/CSS/ComputedValues.h

```diff
@@ -719,7 +719,7 @@ class MutableComputedValues final : public ComputedValues {
         m_inherited = static_cast<MutableComputedValues const&>(other).m_inherited;
     }
 
-    void set_aspect_ratio(AspectRatio aspect_ratio) { m_noninherited.aspect_ratio = aspect_ratio; }
+    void set_aspect_ratio(AspectRatio const& aspect_ratio) { m_noninherited.aspect_ratio = aspect_ratio; }
```

Comment:
We should go the other way - let callers `move` the argument into this function call, and `move` the argument into the class member:
```c++
void set_aspect_ratio(AspectRatio aspect_ratio) { m_noninherited.aspect_ratio = move(aspect_ratio); }
```

Reply:
Got you